cpp/src/parquet/encoding_test.cc

```diff
@@ -495,7 +488,9 @@ class TestArrowBuilderDecoding : public ::testing::Test {
   void CheckDecodeArrowNonNullUsingDenseBuilder() {
     for (auto np : null_probabilities_) {
       InitTestCase(np);
-      SKIP_TEST_IF(null_count_ > 0)
+      if (null_count_ > 0) {
+        GTEST_SKIP();
```

Comment:
This should probably be `continue` instead.

Reply:
Right, done